### ISSUE TITLE
could not use MEM_WRITE characteristic with PE section.

### ISSUE THREAD

**Issue by ParkHanbum** [lief-project/LIEF#115]
```
Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import lief
>>> section = lief.PE.Section('.test')
>>> section
<lief.PE.Section object at 0x00BF7068>
>>> print section
.test     0         0         0         0         0         -0
>>> section.characteristics = lief.PE.SECTION_CHARACTERISTICS.MEM_WRITE | lief.PE.SECTION_CHARACTERISTICS.CNT_INITIALIZED_DATA
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: (): incompatible function arguments. The following argument types are supported:
    1. (arg0: LIEF::PE::Section, arg1: int) -> None

Invoked with: <lief.PE.Section object at 0x00BF7068>, -2147483584L
>>>
```

I have test with 0.7.0.
I guess cause the reason is that in integer overflow.

**Comment by romainthomas** [comment]
Fixed with 0.8.3

**Comment by romainthomas** [comment]
Fixed with 0.8.3
